### ISSUE TITLE
QA and spot check review for Contact information (VAMC Facility health services)

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#14320]
### For reference

**VSA Development Review** - [https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-review-process.md#development-review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/vsa-accessibility-review-process.md#development-review)

## Team Name

Facilities 
(This work was a collaborative effort with CMS and VAMC-upgrade teams.)

## Product/Feature Name

VAMC Facility health services Contact information 

## Product description

**Background information:** The area for review is the front end implementation of a new CMS content structure for VAMC facility health services. The data structure was "hardened" to limit the amount of content which was editable by local content editors. The goal of hardening is use nationally created/controlled content where possible so editing can occur in only one place and cascade throughout the various facilities. Hardening also limits the inadvertent introduction of accessibility errors and makes the content reusable on other pages throughout VA.gov. 

**Scope:** Contact information section within the accordions for University Drive health services on staging: https://staging.va.gov/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/

## Review stage

- [ ] **Development intent review** (Review design to discuss approach to take with development, identify what components and semantic HTML would be appropriate, identify any accessibility issues and how to address) - This would happen as designs are passed to development, before development begins.

- [x] **Spot check review** (Review and follow the steps in the [508 Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md) first.) This happens well before launch, so that the 508 office can review and any issues addressed *before* launch.

- [x] **Other:** QA review

## Point of contact

Michelle Middaugh

## Staging URL and credentials

https://staging.va.gov/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/
no credentials are necessary

## Test plan

1. Visit https://staging.va.gov/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/
2. Navigate to the area on the page titled **Health services offered here**
3. Open the accordions for the various health services

4. Observe each health service has some combination of contact information for _one or more location_, which may include: 

- Name
- Address
- Interactive link to map
- Hours on separate lines by day
- Additional text information about hours (if available)
- One or more phone numbers with label above interactive phone number
- One or more email addresses with label above interactive email address
- Fax number
- Appointments section
- Intro text
- Phone number (optional)
- Referral needed? (optional)
- Walk-ins accepted? (optional)
- Online scheduling (optional)
- Care we provide at VA Pittsburgh health care with text and list
- May contain link to detail page, such as Learn more about....

<details>
<summary> Addiction and substance abuse example </summary>

![spot check facility hs.png](https://images.zenhubusercontent.com/5d8a47393ac3440001d1a2ed/240d6003-3bdc-44fb-a2c7-ac3ffa3606dd)

</details>

**Comment by jenstrickland** [comment]
👋  @mmiddaugh Do you have a needed-by date for the accessibility spot check part of this review? I stopped doing spot checks since my bandwidth is so poor. I can try, and want to be sure I get the review completed within the needed timeframe.

**Comment by tlei123** [comment]
@mmiddaugh , would it be possible to determine a minimum Pass threshold for #4 in your Test Plan above please?

Alternatively, @brangi / @mpelzsherman , maybe one of you can see if the template display logic for any of the contact-info-items are unconditional?

I'm hoping we can find some minimal set of contact-info-items that all facilities' services data should have.

Please let me know.  Thanks much!

**Comment by jenstrickland** [comment]
@mmiddaugh 👋  Rather than document issue tickets for this, I'm noting the issues I found. Please let me know if you would prefer that I create issue tickets. 

It's really just the heading issue repeated throughout (including in the other accordions). The heading order issues noted here would all be [508-defect-2 (⚠️ Serious, to be fixed by 1-2 sprints post-launch at the latest](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2). I noticed a couple of other things in the area I am not supposed to look at (since I opened the wrong set of accordions at first 🤦‍♀️ ), but my guess is those are of scope for this discussion. Let me know if you'd like me to note those, too. 


1. The e2e tests must to be modified to ensure that all of the accordions are opened and the axe checks run on the initially hidden content. There are heading order issues inside those accordions. (I acknowledge that this is not within the area that is the scope of this review.)

![Screen Shot 2020-10-30 at 4 14 01 PM](https://user-images.githubusercontent.com/57469/97753101-9c6cee80-1acb-11eb-920d-761c7e73020d.png)

<hr/>

2. Inside the scope of this review, the same issue is found. So, the first recommendation is to adjust the e2e tests to reveal all hidden content and run the axe checks on it. Another recommendation is to run **axe-coconut** on each accordion's content. I know it's redundant (because I'm doing it now). As you run the scan for each area (really much easier to do this via the e2e test, since it's less monotonous), you'll notice some errors shown. I've included this marked-up image to show what can be ignored.
 
![Screen Shot 2020-10-30 at 4 22 10 PM](https://user-images.githubusercontent.com/57469/97753469-36cd3200-1acc-11eb-8e4f-9d7aa33a6bae.png)

![Screen Shot 2020-10-30 at 4 29 48 PM](https://user-images.githubusercontent.com/57469/97754240-8bbd7800-1acd-11eb-9a0b-82c1e2379b5e.png)

**Comment by mmiddaugh** [comment]
Thanks, @jenstrickland I've created issues for these. 
[#15701](https://github.com/department-of-veterans-affairs/va.gov-team/issues/15701) for headings within the facility health service accordions 
[#15704](https://github.com/department-of-veterans-affairs/va.gov-team/issues/15704) for headings within the "Prepare for your visit" accordions

**Comment by tlei123** [comment]
@mmiddaugh , spot-check done.

Just 1 defect logged.  #16635 for double-spaced addresses.

**Comment by mmiddaugh** [comment]
Heading issues have been resolved
